### ISSUE TITLE
Grids fail on ie11 when dealing with tree data

### ISSUE THREAD

**Issue by nestoralonso** [ag-grid/ag-grid-react#96]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid-react/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Just did a simple tree grid like the one described in https://www.ag-grid.com/javascript-grid-tree/#gsc.tab=0 but using the ag-grid-react library and updating some prop like rowData after the grid has been instantiated. When tested on ie11 it doesn't render the tree like in chrome and in the console throws a warning hinting that an object was not provided to Object.keys 

**Expected behavior**
It should show the expected tree view

**Minimal reproduction of the problem with instructions**

Download https://github.com/nestoralonso/ag-grid-react-ie11-bug and run it in ie11, the problem appears when the componentWillReceiveProps is called for some reason like updating a boolean prop.

**Please tell us about your environment:**

* Win10 with Linux Subsystem
* npm 3.10.10
* node v6.11.2

* **ag-grid-react version:** 13.2

* **ag-Grid/ag-Grid-Enterprise version:** 13.2

* **React version:** 13.2

* **Browser:** Internet Explorer 11

* **Language:** [ES6/7]

**Comment by nestoralonso** [comment]
The problem is in the componentWillReceiveProps of ag-grid-react. Link to a repo to reproduce the issue https://github.com/nestoralonso/ag-grid-react-ie11-bug

**Comment by seanlandsman** [comment]
Hi,

Thanks for raising this. I've raised AG-855 our side to track this - you can view its progress on our pipeline page here: https://www.ag-grid.com/ag-grid-pipeline/

We'll update this thread with updates as soon as we have any.

thanks

**Comment by seanlandsman** [comment]
Hi @nestoralonso 

Thanks for reporting this, and for the repo. The issue was actually much more subtle than just an IE11, so I appreciate you reporting it!

This has now been fixed and will be in our next release.

thanks

**Comment by nestoralonso** [comment]
Nice to know

**Comment by JohnMasterson** [comment]
Just to confirm that this was included in version 14.0.0 which is now available.

Regards, 

John
